frappe/templates/discussions/discussions.js

```diff
@@ -67,26 +67,19 @@ const show_new_topic_modal = (e) => {
 };
 
 const setup_socket_io = () => {
-	const assets = [
-		"/assets/frappe/js/lib/socket.io.min.js",
-		"/assets/frappe/js/frappe/socketio_client.js"
-	];
-
-	frappe.require(assets, () => {
-		if (window.dev_server) {
-			frappe.boot.socketio_port = "9000";
-		}
+	if (window.dev_server) {
+		frappe.boot.socketio_port = "9000";
```

Comment:
"9000" should be a fallback value? There's a key in the (common_)site_config `socketio_port` which should be used first.